### PR TITLE
created list components

### PR DESCRIPTION
**Pull request by JantuDeb**
### List

1. Basic List
2. List with active item
3. List with links
4. Notification list

image
image
image

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.list-item a{
+    color: inherit;
+}
+.list-item a:hover{
+    color: inherit;
+}
+
+
+.list-item.active a{
+    color: #fff;
```
Comment (rkapoor10) [inline components/list.css:34]: use color classes as this is just one line class.

File: documentation/styles.css [hunk 2]
```diff
@@ -110,3 +110,8 @@ header {
 .input-container {
   align-items: flex-start;
 }
+
+/* LIST */
+.list-container .list{
+  max-width: 480px;
```
Comment (rkapoor10) [inline documentation/styles.css:116]: maybe you can use rems  here

**Review by rkapoor10 — APPROVED** [review]
The code is well structured; good to merge.

**Review by ozairkhan2810 — APPROVED** [review]
code looks clean, good to merge

File: components/list.css (hunk 1 above)
Comment (JantuDeb) [reply components/list.css:34]: Thanks, I will that

**Pull request merged**